spec/unit/server_spec.rb

```diff
@@ -61,13 +61,28 @@ def app
 
   describe "show" do
     describe "GET /:id" do
+      let(:description_d) {
+        create_test_description(
+          name: "description_d",
+          store_on_disk: true,
+          scopes: ["os"],
+          format_version: 7
+        )
+      }
+
       it "returns the page" do
         get "/#{description_a.name}"
-
         expect(last_response).to be_ok
         expect(last_response.body).
           to include("#{description_a.name} - Machinery System Description")
       end
+
+      it "returns update instruction page" do
```

Comment:
I would add a context here why an update instruction page is shown and also test the case for newer system descriptions.